### DISCUSSION TITLE
Anyone able to run Mistral-Nemo-Instruct GGUF?

### DISCUSSION THREAD

**Discussion by Tatalebuj** [YellowRoseCx/koboldcpp-rocm#59] [General]
Greetings, 

I've seen lots of people saying how great this new model is, but I keep getting an unhandled exception every time I try.  Has anyone had any luck with this model or one like it?  

v1.70
![image](https://github.com/user-attachments/assets/d4ac792e-fb30-444b-a405-cebca0350746)

v1.67
![image](https://github.com/user-attachments/assets/8362ee6b-3415-4c25-9334-bc46b555942c)

I'm running Windows 11 Pro, AMD 5 7600x, 64gb DDR5 4800 RAM.  Thanks!

**Comment by dmjohn0x** [comment]
Downloading to test now. But it looks like this model was developed in partnership with Nvidia, and I wouldnt put it past them to purposely do something to the model outside what is normal to break ROCm.

Edit: Yes. This LLM in particular does not work. Tried 3 separate quants of this model and none of them worked.

**Comment by dmjohn0x** [comment]
From the Huggingface discussions: 
![image](https://github.com/user-attachments/assets/820e423f-67c8-4be0-9de6-1182dbc541c7)

**Comment by YaFiny** [comment]
In the latest version 1.71, the Mistral-Nemo-Instruct GUF works fine. Thanks to the author of koboldcpp-rocm!!! I tested the Mistral-Nemo and I really liked this model. It fits completely on the Radeon RX 6800 XT and therefore works very fast. The quality of Mistral-Nemo responses is higher than that of Lama 2 by 20 billion parameters.
